cdap-docs/admin-manual/source/installation/index.rst

```diff
@@ -50,25 +50,37 @@ Installation
 - |sdk|_ Installing the CDAP Standalone SDK on Linux, MacOS, and Windows systems. *(Developers' Manual)*
 
 
+.. _modes-data-application-platform:
+
 .. rubric:: Putting CDAP into Production
 
-The Cask Data Application Platform (CDAP) can be run in different modes: in-memory mode
-for unit testing, Standalone CDAP for testing on a developer's laptop, and Distributed
-CDAP for staging and production.
+The Cask Data Application Platform (CDAP) can be run in different modes: **In-memory mode**
+for unit testing and continuous integration pipelines, **Standalone CDAP** for testing on a
+developer's laptop, and **Distributed CDAP** for staging and production.
 
-Regardless of the runtime edition, CDAP is fully functional and the code you develop never
+Regardless of the runtime mode, CDAP is fully-functional and the code you develop never
 changes. However, performance and scale are limited when using in-memory or standalone
-CDAPs.
+CDAPs. CDAP Applications are developed against the CDAP APIs, making the switch between
+these modes seamless. An application developed using a given mode can easily run in
+another mode.
 
 
 .. _in-memory-data-application-platform:
 
 .. rubric:: In-memory CDAP
 
-The in-memory CDAP allows you to easily run CDAP for use in unit tests. In this mode, the
-underlying Big Data infrastructure is emulated using in-memory data structures and there
-is no persistence. The CDAP UI is not available in this mode.
+The in-memory CDAP allows you to easily run CDAP for use in unit tests and continuous
+integration (CI) pipelines. In this mode, the underlying Big Data infrastructure is
+emulated using in-memory data structures and there is no persistence. The CDAP UI is not
+available in this mode. See :ref:`test-cdap` for information and examples on using this
+mode.
+
+- Purpose-built for writing unit tests and CI pipelines
```

Comment:
these points seem sort of abrupt. Do you think they need a section header? Features?